TableSort.svelte

```diff
@@ -57,19 +59,20 @@
                 sortOrder = [[fieldName, 0]]
             }
         }
-        th.className =
-            CLASSNAME_SORTABLE +
-            ' ' +
-            (sortOrder[sortOrder.length - 1][1]
+
+        descAsc = sortOrder[sortOrder.length - 1][1]
                 ? CLASSNAME_DESC
-                : CLASSNAME_ASC)
+                : CLASSNAME_ASC
+
+        th.classList.add('CLASSNAME_SORTABLE')
+        th.classList.add(descAsc)
     }
 
     function resetClasses() {
         const th = thead.getElementsByTagName('th')
         for (let i = 0; i < th.length; i++) {
             if (th[i].dataset.sort) {
-                th[i].className = CLASSNAME_SORTABLE
+                th[i].classList.add(CLASSNAME_SORTABLE)
```

Comment:
Add the below to clear any leftover "ascending" or "descending" classes where there are no longer being sorted.
```
if (th[i].classList.contains(CLASSNAME_ASC)) {
          th[i].classList.remove(CLASSNAME_ASC);
        } else if (th[i].classList.contains(CLASSNAME_DESC)) {
          th[i].classList.remove(CLASSNAME_DESC);
        }
```